infrastructure/logging/src/testFixtures/resources/log4j2-test.xml

```diff
@@ -8,6 +8,7 @@
     </Appenders>
     <Loggers>
         <Logger name="io.netty" level="INFO"/>
+        <Logger name="com.github.dockerjava.zerodep.shaded.org.apache.hc.client5.http.wire" level="INFO"/>
```

Comment:
This was ridiculously noisy and meant the test output exceeded the maximum size CircleCI would parse to show test results...